WordPress/src/main/java/org/wordpress/android/ui/reader/subfilter/SubfilterListItem.kt

```diff
@@ -23,7 +23,7 @@ sealed class SubfilterListItem(val type: ItemType) {
         return if (type == otherItem.type) {
             when (type) {
                 SECTION_TITLE -> label == otherItem.label
-                SITE -> (this as Site).blog.isSameAs((otherItem as Site).blog)
+                SITE -> (this as Site).blog.isSameAs((otherItem as Site).blog, false)
```

Comment:
Do we want to find out:
a) the two objects are referencing the same blog
b) the two objects are equals

If the answer is `a`, can't we just compare their ids?
Wdyt?

Reply:
so , we want `a`. This said at a gut level I like more the multifield match 😊. Jokes apart at a logical level I agree the ids should be enough, not have enough context to say if that is really unique among blogs/feeds. I think the multifield comparison should be safe but I'm sure you have a deeper understanding of the overall architecture/infrastructure so I'm happy to change if you think it's more readable/efficient (and works of course 😬).

Reply:
I don't have enough context either. It just feels very weird to be comparing equality when we want to mark a blog as selected if it matches the selected blog. But if you copied it from the other part of the reader, let's just leave it as it is.

SideNote: Even more weird is that the method is called `isSameAs` even though it's not `isSame` comparison but `isEquals` comparison :D. But I understand you didn't introduce it.